regression/pages/whitelabel/home_page.py

```diff
@@ -107,3 +107,9 @@ def go_to_registration_page(self):
         Go to registration page
         """
         self.q(css='.btn-brand.btn-client[href="/register"]').click()
+
+    def go_to_courses_page(self):
+        """
+        Go to courses page
+        """
+        self.q(css='.brand-link[href="/courses"]').click()
```

Comment:
Waiting for the page in the [test](https://github.com/edx/edx-e2e-tests/pull/181/files#diff-1ace6cae9636a71d17f88c7d3df4641dR57) actually. Does that make sense?